proctor-common/src/test/java/com/indeed/proctor/common/TestProctor.java

```diff
@@ -295,13 +295,23 @@ public void testDetermineTestGroupsForRandomTestWithRandomEnabled() {
         final Allocation allocation = new Allocation();
         final TestChooser.Result result = new TestChooser.Result(testBucket, allocation);
 
+        final ConsumableTestDefinition testDefinition =
+                ConsumableTestDefinition.fromTestDefinition(
+                        TestDefinition.builder()
+                                .setEnableUnitlessAllocations(true)
+                                .setSalt("&X")
+                                .setTestType(TestType.AUTHENTICATED_USER)
```

Comment:
The test name is " testDetermineTestGroupsFor**Random**TestWithRandomEnabled". Is it meant to test TestType.RANDOM ?

Reply:
Oops did not mean to commit this. Reverted